App/Containers/SplashScreen/SplashScreen.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Text, View,Button } from 'react-native'
+import styles from './SplashScreenStyle'
+import { connect } from 'react-redux'
+import ExampleActions from 'App/Stores/Authentication/Actions'
+import { PropTypes } from 'prop-types'
+
+class SplashScreen extends React.Component {
+  render() {
+    this.props.fetchUser()
```

Comment:
We never call a function which manage state from inside the render function, such actions can somewhere lead to infinite loops.

Reply:
I have Fixed this issue, Please review my pull request once again!